pkg/operator/admission/init.go

```diff
@@ -0,0 +1,175 @@
+package admission
+
+import (
+	"github.com/pkg/errors"
+	"github.com/rook/rook/pkg/clusterd"
+	"github.com/rook/rook/pkg/operator/k8sutil"
+	v1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+	"os"
+)
+
+const (
+	containerName      = "rook-ceph-operator"
+	secretName         = "rook-admission-controller-secret"
+	acServiceName      = "rook-admission-controller-service"
+	webhookConfigName  = "rook-ceph-webhook"
+	kubeNamespace      = "kube-system"
+	configMapName      = "extension-apiserver-authentication"
+	secretVolumeName   = "webhook-certs"
+	secretMountPath    = "/etc/webhook"
+	serviceAccountName = "rook-ceph-system"
```

Comment:
The admission controller shouldn't use the same service account as the operator. It really shouldn't need all the privileges of the operator. First see if you can use the default service account (don't set it). If the permissions are not sufficient, let's create a new service account in common.yaml.